database/mssql/scripts/versions/v_32_ddl.sql

```diff
@@ -0,0 +1,194 @@
+SET ANSI_NULLS ON
+GO
+SET QUOTED_IDENTIFIER ON
+GO
+SET NOCOUNT ON
+GO
+
+SET XACT_ABORT ON
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+--------------------------------------
+-- Holiday table creation
+--------------------------------------
+CREATE TABLE [dbo].[ORBC_HOLIDAY](
+	[ID] [int] IDENTITY(1,1) NOT NULL,
+	[HOLIDAY_YEAR] [int] NOT NULL,
```

Comment:
Discussed with @john-fletcher-aot. We can use a single column called Holiday of type DATE instead of separate column for year, month and day